schema/workflow.yaml

```diff
@@ -430,6 +430,36 @@ $defs:
                 description: Specifies whether redirection status codes (`300–399`) should be treated as errors.
             required: [ document, operationId ]
             unevaluatedProperties: false
+      - title: CallA2A
+        description: Defines the A2A call to perform.
+        $ref: '#/$defs/taskBase'
+        type: object
+        unevaluatedProperties: false
+        required: [ call, with ]
+        properties:
+          call:
+            type: string
+            const: a2a
+          with:
+            type: object
+            title: A2AArguments
+            description: The A2A call arguments.
+            properties:
+              agentCard:
```

Comment:
I think this type must be defined or imported from the a2a.

Reply:
Yes indeed, this discussion can be closed. The `agentCard` is better well-defined as an endpoint or expression.